lab-gary/test/greet-test.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const greet = require('../lib/greet.js');
+const assert = require('assert');
+
+describe('Greet Module', function() {
+  describe('#sayHi', function() {
+    it('should return Hello Gary!', function() {
+      var result = greet.sayHi('Gary');
+      assert.ok(result === 'Hello Gary!', 'not equal to Hello Gary!');
+    });
+
+    it('should throw a missing name error', function() {
+      assert.throws(function() {
+        greet.sayHi();
+      }, 'error not thrown');
+    });
+  });
+  describe('#sayBye', function() {
+    it('should return See you later!', function() {
+      var bye = 'See you later!';
+      assert.ok(bye === 'See you later!', 'not equal to See you later!');
+    });
+  });
+  // attempt to test my command line argument, not sure how to do it.
```

Comment:
You would probably want to use process.argv here in your test and make sure the user input from the CLI lines up here. You'd need to basically manually set the name here into the process.argv array to be used in the function when it's called.